Assets/Scripts/CoinSpawner.cs

```diff
@@ -79,10 +79,15 @@ private IEnumerator RespawnTimer()
             if (SecondsToRespawn >= 0)
             {
                 yield return new WaitForSeconds(SecondsToRespawn);
-                mPhotonView.RPC("Respawn", PhotonTargets.All);
+                RespawnOverNetwork();
             }
         }
 
+        public void RespawnOverNetwork()
+        {
+            mPhotonView.RPC("Respawn", PhotonTargets.All);

```

Comment:
This does not need to be a buffered RPC, correct? I think it would only need to be buffered if despawns were.